### DISCUSSION TITLE
Tags/Keywords for collection object records

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5633] [Feature Requests]
In other services like Tropicos, collections can tag a specimen or group of specimens.

It would be nice to have this feature in 7! It definitely would help for project management.

This should be able to be applied in query results.

**Requested By:** NMC Herbaria

**Comment by maxpatiiuk** [comment]
Does this sound like what they need?
A text field on the CO form is used to serve as the `tag` field.
The `tag` field is assigned a pick list of available tags
The pick list system is expanded to allow selecting multiple items.

**Comment by grantfitzsimmons** [comment]
I think that's a fair implementation of this feature request. It might be worth creating a unique system for this feature so it can be used across all tables- maybe independent from the record?

Your idea should be enough for NMC
